app/dashboard/models.py

```diff
@@ -2087,61 +2086,70 @@ def get_funder_leaderboard_index(self):
     def get_org_leaderboard_index(self):
         return self.get_leaderboard_index('quarterly_orgs')
 
-    def get_eth_sum(self, sum_type='collected', network='mainnet'):
+    def get_eth_sum(self, sum_type='collected', network='mainnet', bounties=None):
         """Get the sum of collected or funded ETH based on the provided type.
 
         Args:
             sum_type (str): The sum to lookup.  Defaults to: collected.
             network (str): The network to query results for.
                 Defaults to: mainnet.
+            bounties (dashboard.models.BountyQuerySet): Override the BountyQuerySet this function processes.
+                Defaults to: None.
 
         Returns:
             float: The total sum of all ETH of the provided type.
 
         """
         eth_sum = 0
 
+        if not bounties:
+            if sum_type == 'funded':
+                bounties = self.get_funded_bounties(network=network)
+            elif sum_type == 'collected':
+                bounties = self.get_fulfilled_bounties(network=network)
+            elif sum_type == 'org':
+                bounties = self.get_orgs_bounties(network=network)
+
         if sum_type == 'funded':
-            obj = self.get_funded_bounties(network=network).has_funds()
-        elif sum_type == 'collected':
-            obj = self.get_fulfilled_bounties(network=network)
-        elif sum_type == 'org':
-            obj = self.get_orgs_bounties(network=network)
+            bounties = bounties.has_funds()
 
         try:
-            if obj.exists():
-                eth_sum = obj.aggregate(
+            if bounties.exists():
+                eth_sum = bounties.aggregate(
                     Sum('value_in_eth')
                 )['value_in_eth__sum'] / 10**18
         except Exception:
             pass
 
         return eth_sum
 
-    def get_who_works_with(self, work_type='collected', network='mainnet'):
+    def get_who_works_with(self, work_type='collected', network='mainnet', bounties=None):
         """Get an array of profiles that this user works with.
 
         Args:
             work_type (str): The work type to lookup.  Defaults to: collected.
             network (str): The network to query results for.
                 Defaults to: mainnet.
+            bounties (dashboard.models.BountyQuerySet): Override the BountyQuerySet this function processes.
+                Defaults to: None.
 
         Returns:
             dict: list of the profiles that were worked with (key) and the number of times they occured
 
         """
-        if work_type == 'funded':
-            obj = self.bounties_funded.filter(network=network)
-        elif work_type == 'collected':
-            obj = self.get_fulfilled_bounties(network=network)
-        elif work_type == 'org':
-            obj = self.get_orgs_bounties(network=network)
+        if not bounties:
+            if work_type == 'funded':
+                bounties = self.bounties_funded.filter(network=network)
```

Comment:
The renaming of these vars makes review a little bit more difficult with all of the additional line changes.  I'm fine with it, just something to consider in the future.